.github/workflows/lcm_dreamshaper_cpp.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         run: |
           conda activate openvino_lcm_cpp
           conda update -c conda-forge --all
-          conda install -c conda-forge openvino=2024.1.0 c-compiler cxx-compiler git make cmake
+          conda install -c conda-forge -c conda-forge/label/openvino_dev openvino==2024.2.0.dev20240513 c-compiler cxx-compiler git make cmake
```

Comment:
```suggestion
          conda install -c conda-forge -c conda-forge/label/openvino_dev openvino c-compiler cxx-compiler git make cmake
```

Reply:
That would select conda-forge/win-64::libopenvino-2024.1.0-hfe1841e_7